packages/video_player/video_player/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: video_player
 description: Flutter plugin for displaying inline video with other Flutter
   widgets on Android and iOS.
-version: 0.10.7
+version: 0.10.8
 homepage: https://github.com/flutter/plugins/tree/master/packages/video_player/video_player
 
 flutter:
```

Comment:
Can we add restrictions on line 36? It crashes on compile if users accidentally flutter pub upgrade otherwise if they had ^0.10.7 right?

Reply:
Nope, my change to the plugins api is backwards compatible with respect to compiling, it just doesn't function unless you have v1.15.3.  Would we prefer to force the upgrade and increment the minor version to 0.11?